tests/unit/test_helpers.py

```diff
@@ -195,6 +196,19 @@ def test_get_os(self):
         self.assertEqual(get_os(False), 'Unknown')
         self.assertEqual(get_os(None), 'Unknown')
 
+    def test_get_str_value(self):
+        """Test getting an expected string, including Python to js-style
+        boolean.
+
+        Note: key (output) and value (input) are in an unexpected order,
+        just to keep the data structure valid.
+        """
+        tests = [{'1': 1}, {'null': None}, {'true': True}, {'false': False},
+                 {'': ''}, {'cool': 'cool'}, {u'\U0001f480': u'💀'}]
+        for test in tests:
+            for output, _input in test.iteritems():
+                self.assertEqual(get_str_value(_input), output)
```

Comment:
I would also rename `_input` (that you call like this to avoid the conflict with the python method) probably as `browser_input` or `user_input`.

Reply:
Thanks, will rename to `browser_input`, it looks less weird than `_input`.